pynest/nest/tests/test_aeif_lsodar.py

```diff
@@ -323,15 +342,55 @@ def test_iaf_behaviour(self):
                 {model: multimeters[model]},
                 aeif_DT0,
                 ref_data,
-                recordables[key])
+                recordables[syn_type])
             self.assert_pass_tolerance(rel_diff, di_tolerances_iaf)
 
+    def test_iaf_dc_current(self):
+        '''
+        The models should behave as iaf_cond_* if a == 0., b == 0. and
```

Comment:
Thanks for adding  a new test. Could you update the docstrings? I realized (and this is my fault) that it should be ``iaf_*`` and not ``iaf_cond_*`` anymore. Also it would be better if you could precise that the first test (``test_iaf_behaviour``) looks at the response to spikes, the second at the response to a dc current